helm/aws-operator/Chart.yaml

```diff
@@ -1,2 +1,3 @@
 name: aws-operator
 version: [[ .Version ]]
+appVersion: [[ .Version ]]
```

Comment:
I don't follow why we need this when it has the same value.

Reply:
We don't necessarily need it as-is because like you say they're the same value, but I'm not sure if they should be. I have a notion we'll need to extract the static version string from `project.go` and use it here and in the labels instead of the git-generated version. I've asked my team to clarify.

Reply:
Hey @mig4, the appVersion should be removed. We use this when packaging upstream components.

So we have our version and the component version. e.g. https://github.com/giantswarm/cert-manager-app/blob/master/helm/cert-manager-app/Chart.yaml#L4

For our stuff we can just use the version field.

Reply:
I think `appVersion` is not misused here but we need to somehow extract the value from `project.go` here. In case of operator chart version and operator version are different. 

E.g. operator in version `8.1.1-dev` can be packaged to chart `operator-8.1.1-sha1` and `operator-8.1.1-sha2`. Only for the released version both operator and chart version must match.

@rossf7 do you think that would be a correct usage?

Reply:
Yes if the versions can be different and `appVersion` provides useful info then totally fine with using it.

Reply:
Done now, replaced value with `AppVersion` which `architect` will need to inject.